### ISSUE TITLE
Official Nook support

### ISSUE THREAD

**Issue by JayPanoz** [FriendsOfEpub/Blitz#3]
Nook is not specifically supported at the moment but it’s a “popular” platform in North America. 

Unfortunately, we don’t have easy access to Nook in Europe so inputs from north american eBook producers is a must. 

[edit] won't cripple the framework to get around their “yolo overrides” though.

**Comment by Quanthir** [comment]
+1 for that. I also think this framework needs Nook support

**Comment by JayPanoz** [comment]
With some research, I've found out that Nook decided to enforce user settings by any means necessary. Source: https://www.mobileread.com/forums/showpost.php?p=1927444&postcount=6 

In other words, thanks to InDesign’s and other authoring software’s CSS output, Nook overrides the CSS selectors at any level of specificity so it would be useless to spend time trying to support Nook.

I'm not even mad at Nook’s software engineers because I agree a lot of tools are abusing CSS and do believe this is an issue. 

To sum things up, this isn't a wontfix, it's a cantfix.
